courses/management/commands/manage_program_certificates.py

```diff
@@ -0,0 +1,75 @@
+"""
+Management command to sync grades and certificates for a course run
+"""
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+
+from courses.models import ProgramEnrollment
+from courses.utils import generate_program_certificate
+from users.api import fetch_user
+
+
+class Command(BaseCommand):
+    """
+    Command to create program certificates for users.
+    """
+
+    help = "Create program certifificates, specific or all, for user(s)."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--user",
+            type=str,
+            help="The id, email or username of the enrolled User",
+            required=False,
+        )
+        parser.add_argument(
+            "--readable_id",
```

Comment:
could you name this as a program? It would still take the readable id but would be more user-friendly. i.g. Take a look at how we are taking course run as in put to other commands https://github.com/mitodl/mitxpro/blob/master/courses/management/commands/sync_courseruns.py